datafusion/core/tests/fuzz_cases/cross_join_fuzz.rs

```diff
@@ -0,0 +1,174 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+//! Fuzz Test for CrossJoinExec
+
+use std::sync::Arc;
+
+use arrow::compute::concat_batches;
+use arrow_array::{ArrayRef, Int32Array, PrimitiveArray, RecordBatch};
+use arrow_schema::{DataType, Field, Schema};
+use datafusion::execution::context::SessionContext;
+use datafusion_common::utils::get_arrayref_at_indices;
+use datafusion_common::{JoinType, Result};
+use datafusion_execution::config::SessionConfig;
+use datafusion_physical_plan::joins::utils::build_join_schema;
+use datafusion_physical_plan::{collect, joins::CrossJoinExec, memory::MemoryExec};
+
+use rand::{rngs::ThreadRng, Rng};
+
+#[tokio::test]
+async fn test_cross_fuzz() {
+    for _ in 0..50 {
```

Comment:
This is not a good way to test it. Pick 5 seeds and make the random generator use that.